README.md

```diff
@@ -1,10 +1,31 @@
-# Blockchain contracts
+<a href="https://comit.network">
+<img src="logo.svg" height="120px">
+</a>
+
+---
+
+[COMIT](https://comit.network) is an open protocol facilitating cross-blockchain applications.
+For example, with [COMIT](https://comit.network) you can exchange Bitcoin for Ether or any ERC20 token directly with another person.
+
+This repository contains the the assembly/script code of blockchain contracts used in the COMIT protocol.
```

Comment:
🔧 
```suggestion
This repository contains the assembly/script code of blockchain contracts used in the COMIT protocol.
```

Reply:
please add `no-mergify` label if you mark such kind of things :)